open-api/rest-catalog-open-api.yaml

```diff
@@ -4574,6 +4634,23 @@ components:
               }
             }
 
+    IdempotencyInProgressResponse:
```

Comment:
Are we referencing this response some where ?

Reply:
You are right: It's not referenced.
I reconsidered the approach after reviewing the spec. I changed from a `Response` component to an `Example` component because it's the correct pattern for handling multiple conflict types under the same 409 status code.